frontend/src/bundles/common/components/select/select.tsx

```diff
@@ -39,6 +42,8 @@ const Select = <T extends FieldValues>({
     placeholder = 'Placeholder',
 }: Properties<T>): JSX.Element => {
     const firstElementIndex = 0;
+    const secondElementIndex = 1;
```

Comment:
Maybe this should be in capital letters, [B22](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md)

Reply:
Thanks